cmd/contractWatcher.go

```diff
@@ -103,18 +105,18 @@ func contractWatcher() {
 	case "full":
 		t = ft.NewTransformer(con, blockChain, &db)
 	default:
-		log.Fatal("Invalid mode")
+		LogWithCommand.Fatal("Invalid mode")
 	}
 
 	err := t.Init()
 	if err != nil {
-		log.Fatal(fmt.Sprintf("Failed to initialized transformer\r\nerr: %v\r\n", err))
+		LogWithCommand.Fatal(fmt.Sprintf("Failed to initialized transformer err: %v ", err))
```

Comment:
:%s/initialized/initialize

also wonder if we want a `,` before `err:`

Reply:
grammar for the win 🥇